pkg/storj/node_test.go

```diff
@@ -140,6 +141,24 @@ func TestNodeID_String_Version(t *testing.T) {
 	}
 }
 
+func TestNodeID_MarshalJSON(t *testing.T) {
+	nodeID, _ := storj.NodeIDFromString("12vha9oTFnerxYRgeQ2BZqoFrLrnmmf5UWTCY2jA77dF3YvWew7")
+	buf, err := json.Marshal(nodeID)
+	if !assert.NoError(t, err) {
+		t.Fatal()
+	}
```

Comment:
You can simplify this check by using `require.NoError(t, err)`. It will fail the test immediately if there is an error.